app/src/js/styles/navigation_page.tsx

```diff
@@ -35,5 +36,5 @@ export const dividedPageStyle = (theme: Theme) => createStyles({
     flexGrow: 1
   },
 
-  appBarSpacer: theme.mixins.toolbar
+  appBarSpacer: theme.mixins.toolbar as CSSProperties
```

Comment:
Why do we need type coercion here? I know it fixes the compilation errors. But it is helpful to document here why the types don't match and why this hack is necessary.

Reply:
Honestly, I don't know. CSSProperties is one of the accepted types, but it looks like typescript is trying to match against another one of the unioned types (see below). Looks like it may be a typescript issue, but I don't know for sure, and I'm not sure how to find out.
![image](https://user-images.githubusercontent.com/7757242/73030421-786ad700-3dee-11ea-8924-16af174fb926.png)